[receiver/sqlquery] Flaky Test - TestOracleDBIntegration

Seen in https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/7260768208?check_suite_focus=true

```
=== RUN   TestOracleDBIntegration
2022/07/09 02:42:12 Starting container id: 04efba7ebdc1 image: 48c01a27-940c-4c81-af6f-a024450bc152:05ed71c7-559f-4da4-92e8-9735caad8a95
2022/07/09 02:42:12 Waiting for container id 04efba7ebdc1 image: 48c01a27-940c-4c81-af6f-a024450bc152:05ed71c7-559f-4da4-92e8-9735caad8a95
2022/07/09 02:43:13 Container is ready id: 04efba7ebdc1 image: 48c01a27-940c-4c81-af6f-a024450bc152:05ed71c7-559f-4da4-92e8-9735caad8a95
coverage: 55.3% of statements
panic: test timed out after 5m0s
goroutine 149 [running]:
testing.(*M).startAlarm.func1()
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1788 +0xbb
created by time.goFunc
	/opt/hostedtoolcache/go/1.17.11/x64/src/time/sleep.go:180 +0x4a
goroutine 1 [chan receive, 4 minutes]:
testing.(*T).Run(0xc00018a9c0, {0x1a31d18, 0x17}, 0x1aa8070)
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1307 +0x752
testing.runTests.func1(0x0)
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1598 +0x9a
testing.tRunner(0xc00018a9c0, 0xc0005bfbc0)
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1259 +0x230
testing.runTests(0xc00019c080, {0x2a8cca0, 0x13, 0x13}, {0xc0000415a8, 0xc0005bfc48, 0x2aa00a0})
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1596 +0x7cb
testing.(*M).Run(0xc00019c080)
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1504 +0x9d2
main.main()
	_testmain.go:137 +0x325
goroutine 21 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00019c100)
	/home/runner/go/pkg/mod/go.opencensus.io@v0.23.0/stats/view/worker.go:276 +0x185
created by go.opencensus.io/stats/view.init.0
	/home/runner/go/pkg/mod/go.opencensus.io@v0.23.0/stats/view/worker.go:34 +0xf9
goroutine 117 [select]:
github.com/stretchr/testify/assert.Eventually({0x1cc26a0, 0xc000640340}, 0xc000307430, 0x43d84d, 0xc000987ec8, {0xc000987248, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/assert/assertions.go:1744 +0x2aa
github.com/stretchr/testify/assert.Eventuallyf({0x1cc26a0, 0xc000640340}, 0x0, 0x1cc26a0, 0xc000307430, {0x1a469df, 0x25}, {0x0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/assert/assertion_format.go:155 +0x18e
github.com/stretchr/testify/require.Eventuallyf({0x1cd4808, 0xc000640340}, 0x0, 0x0, 0x7, {0x1a469df, 0x25}, {0x0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/require/require.go:375 +0xde
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlqueryreceiver.TestOracleDBIntegration(0xc000640340)
	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/sqlqueryreceiver/integration_test.go:224 +0xc65
testing.tRunner(0xc000640340, 0x1aa8070)
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1259 +0x230
created by testing.(*T).Run
	/opt/hostedtoolcache/go/1.17.11/x64/src/testing/testing.go:1306 +0x727
goroutine 15 [chan receive, 4 minutes]:
github.com/testcontainers/testcontainers-go.(*Reaper).Connect.func1({0x1cfe950, 0xc000010108})
	/home/runner/go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.13.0/reaper.go:134 +0x876
created by github.com/testcontainers/testcontainers-go.(*Reaper).Connect
	/home/runner/go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.13.0/reaper.go:106 +0x2fd
goroutine 105 [select, 4 minutes]:
database/sql.(*DB).connectionOpener(0xc0001901a0, {0x1ceb7a0, 0xc0005ae8c0})
	/opt/hostedtoolcache/go/1.17.11/x64/src/database/sql/sql.go:1196 +0xfa
created by database/sql.OpenDB
	/opt/hostedtoolcache/go/1.17.11/x64/src/database/sql/sql.go:794 +0x307
goroutine 106 [select]:
go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1()
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.55.0/receiver/scraperhelper/scrapercontroller.go:170 +0x236
bufio.(*Reader).Peek(0xc00063a[900](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/7260768208?check_suite_focus=true#step:5:901), 0x1)
	/opt/hostedtoolcache/go/1.17.11/x64/src/bufio/bufio.go:139 +0xcc
net/http.(*persistConn).readLoop(0xc0000f45a0)
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:2087 +0x2db
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:1747 +0x2ef8
goroutine 90 [select, 4 minutes]:
net/http.(*persistConn).writeLoop(0xc0000f4360)
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:2386 +0x1a9
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:1748 +0x2f8b
goroutine 120 [select]:
net/http.(*persistConn).writeLoop(0xc0000f45a0)
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:2386 +0x1a9
created by net/http.(*Transport).dialConn
	/opt/hostedtoolcache/go/1.17.11/x64/src/net/http/transport.go:1748 +0x2f8b
goroutine 125 [chan receive, 4 minutes]:
github.com/testcontainers/testcontainers-go.(*Reaper).Connect.func1({0x1cfe950, 0xc000010[918](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/7260768208?check_suite_focus=true#step:5:919)})
	/home/runner/go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.13.0/reaper.go:134 +0x876
created by github.com/testcontainers/testcontainers-go.(*Reaper).Connect
	/home/runner/go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.13.0/reaper.go:106 +0x2fd
goroutine 132 [select]:
database/sql.(*DB).connectionOpener(0xc00084f1e0, {0x1ceb7a0, 0xc000828f00})
	/opt/hostedtoolcache/go/1.17.11/x64/src/database/sql/sql.go:1196 +0xfa
created by database/sql.OpenDB
	/opt/hostedtoolcache/go/1.17.11/x64/src/database/sql/sql.go:794 +0x307
goroutine 133 [select]:
go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1()
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.55.0/receiver/scraperhelper/scrapercontroller.go:170 +0x236
created by go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping
	/home/runner/go/pkg/mod/go.opentelemetry.io/collector@v0.55.0/receiver/scraperhelper/scrapercontroller.go:161 +0x91
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlqueryreceiver	300.148s
FAIL
```

--- a/Makefile.Common
+++ b/Makefile.Common
@@ -4,7 +4,7 @@ SRC_ROOT := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -v -timeout 300s --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 300s
+GOTEST_INTEGRATION_OPT?= -race -timeout 360s
 GOTEST_OPT_WITH_COVERAGE = $(GOTEST_OPT) -coverprofile=coverage.txt -covermode=atomic
 GOTEST_OPT_WITH_INTEGRATION=$(GOTEST_INTEGRATION_OPT) -v -tags=integration,$(GO_BUILD_TAGS) -run=Integration -coverprofile=integration-coverage.txt -covermode=atomic
 GOCMD?= go

